packages/react/src/index.ts

```diff
@@ -120,6 +120,7 @@ export {
   ColumnList
 } from './components/TwoColumnPanel';
 export { default as Notice } from './components/Notice';
+export { default as Popover, PopoverProps } from './components/Popover';
```

Comment:
We don't typically export props here.

```suggestion
export { default as Popover } from './components/Popover';
```